okonomiyaki/file_formats/_package_info.py

```diff
@@ -220,7 +240,11 @@ def __init__(self, metadata_version, name, version, platforms=None,
         self.requires_dist = requires_dist or ()
         self.provides_dist = provides_dist or ()
         self.obsoletes_dist = obsoletes_dist or ()
-        self.project_urls = projects_urls or ()
+        self.project_urls = project_urls or ()
+
+        # version 2.1
+        self.description_content_type = description_content_type or ""
+        self.provides_extra = provides_extra or ()
 
     def to_string(self, metadata_version_info=MAX_SUPPORTED_VERSION):
```

Comment:
@kihneman We need to update the to_string method to be aware of PKG_INFO version 2.1